### DISCUSSION TITLE
GNOME style task swapping for applications from the dock

### DISCUSSION THREAD

**Discussion by rowanfr** [pop-os/cosmic-epoch#1421] [Ideas]
It would be useful to my current mental workflow if we had an option to swap between applications by using `Super+#` such as `Super+2` to swap between or startup different apps set from the center dock instead of using `Ctrl+#` after we bring up the Launcher. This might also be present already due to this Desktop Environment being based on GNOME and if that is the case and I simply haven't found how to configure this please let me know.

**Comment by chryan** [comment]
Adding context for how this feature would behave:

- Window only has 1 instance
    - If window is unfocused, focus it (and restore if minimized)
    - If window is focused, minimize it
- Window has more than 1 instance (i.e. multiple browser windows)
    - Bring up the multiple window preview
    - While holding and not letting go of SUPER, tapping the 1 will cycle through the window instances, releasing keys will focus on the window that was cycled to
